### ISSUE TITLE
[Issue] Tried to load unspecified class: HTTP::Cookie when running fastlane spaceauth -u

### ISSUE THREAD

**Issue by dimmdesign** [fastlane/fastlane#8011]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

**`Tried to load unspecified class: HTTP::Cookie`**
```
/Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/class_loader.rb:97:in `find': [!] Tried to load unspecified class: HTTP::Cookie (Psych::DisallowedClass)
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/class_loader.rb:28:in `load'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/to_ruby.rb:396:in `resolve_class'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/to_ruby.rb:208:in `visit_Psych_Nodes_Mapping'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/visitor.rb:16:in `visit'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/visitor.rb:6:in `accept'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/to_ruby.rb:32:in `accept'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/to_ruby.rb:330:in `block in register_empty'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/to_ruby.rb:330:in `each'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/to_ruby.rb:330:in `register_empty'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/to_ruby.rb:141:in `visit_Psych_Nodes_Sequence'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/visitor.rb:16:in `visit'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/visitor.rb:6:in `accept'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/to_ruby.rb:32:in `accept'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/to_ruby.rb:311:in `visit_Psych_Nodes_Document'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/visitor.rb:16:in `visit'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/visitor.rb:6:in `accept'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych/visitors/to_ruby.rb:32:in `accept'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/2.3.0/psych.rb:302:in `safe_load'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/fastlane-2.11.0/spaceship/lib/spaceship/spaceauth_runner.rb:36:in `run'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/fastlane-2.11.0/spaceship/lib/spaceship/commands_generator.rb:40:in `block (2 levels) in run'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/command.rb:178:in `call'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/command.rb:153:in `run'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/runner.rb:446:in `run_active_command'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/fastlane-2.11.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:38:in `run!'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/commander-4.4.3/lib/commander/delegates.rb:15:in `run!'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/fastlane-2.11.0/spaceship/lib/spaceship/commands_generator.rb:46:in `run'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/fastlane-2.11.0/spaceship/lib/spaceship/commands_generator.rb:12:in `start'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/fastlane-2.11.0/fastlane/lib/fastlane/cli_tools_distributor.rb:45:in `take_off'
	from /Users/jenkins/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/fastlane-2.11.0/bin/fastlane:15:in `<top (required)>'
	from /Users/jenkins/.rbenv/versions/2.3.1/bin/fastlane:23:in `load'
	from /Users/jenkins/.rbenv/versions/2.3.1/bin/fastlane:23:in `<main>'
```

### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                 | Value                                       |
| ------------------- | ------------------------------------------- |
| OS                  | 10.12                                       |
| Ruby                | 2.3.1                                       |
| Bundler?            | false                                       |
| Git                 | git version 2.7.4 (Apple Git-66)            |
| Installation Source | ~/.rbenv/versions/2.3.1/bin/fastlane        |
| Host                | Mac OS X 10.12 (16A323)                     |
| Ruby Lib Dir        | ~/.rbenv/versions/2.3.1/lib                 |
| OpenSSL Version     | OpenSSL 1.0.2j  26 Sep 2016                 |
| Is contained        | false                                       |
| Xcode Path          | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version       | 7.3.1                                       |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_GB.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.11.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                | Version | Update-Status |
| --------------------- | ------- | ------------- |
| fastlane-plugin-badge | 0.7.0   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.0.0        |
| slack-notifier            | 1.5.1        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.4        |
| terminal-notifier         | 1.7.1        |
| plist                     | 3.2.0        |
| addressable               | 2.5.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| excon                     | 0.54.0       |
| unf_ext                   | 0.0.7.2      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20161129 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| highline                  | 1.7.8        |
| mini_magick               | 4.5.1        |
| multi_json                | 1.12.1       |
| multi_xml                 | 0.6.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| bundler                   | 1.13.7       |
| i18n                      | 0.7.0        |
| thread_safe               | 0.3.5        |
| tzinfo                    | 1.2.2        |
| minitest                  | 5.10.1       |
| activesupport             | 4.2.7.1      |
| json                      | 2.0.2        |
| commander                 | 4.4.3        |
| io-console                | 0.4.5        |
| rubyzip                   | 1.2.0        |
| CFPropertyList            | 2.3.4        |
| claide                    | 1.0.1        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.4.2        |
| terminal-table            | 1.7.3        |
| unicode-display_width     | 1.1.3        |
| badge                     | 0.7.1        |
| fastlane-plugin-badge     | 0.7.0        |
</details>

*generated on:* **2017-01-26**
</details>

**Comment by hjanuschka** [comment]
@dimmdesign thank you for reporting the issue, could you try out this PR -> https://github.com/fastlane/fastlane/pull/8012 and let me know if this fixes the problem for you?

**Comment by dimmdesign** [comment]
@hjanuschka The same issue appears even using the source from your PR #8012 
I built Fastlane source code based on the description on the `YourFirstPR.md` with no luck, issue remains.

**Comment by KrauseFx** [comment]
@dimmdesign would you mind jumping into the discussion over there? https://github.com/fastlane/fastlane/issues/8012

**Comment by mfurtak** [comment]
Thank for for the report! 👍 This should be addressed by #8012 which will go out in the next release 🚀
